### DISCUSSION TITLE
Can Sharepoint Modern Pages with Lists-Document Library and PnP Search be Google Searchable

### DISCUSSION THREAD

**Discussion by RamBalani** [microsoft-search/pnp-modern-search#2078] [Q&A] [answered]
We have a Sharepoint portal site where external guests accounts are enabled, i.e. Guest Account in Azure Active Directory.
External Guest are share the portal site landing page menu--visitors click--retrieve a one-time pass code Microsoft generates in the guest emails outside Sharpoint, .e.g. Yahoo email account--Sharepoint Online allows access to them as Site Visitors.

Can these portal site pages be reachable by Google ? We are interested in exposing them  so SEO is boosted but not allow anonymous access obviously.
Any help or information would be greatly  appreciated.

I believe Sharepoint Powerapps allows anonymous portal creation but all we do expose Modern Page Search including a PnP Search SPFx from Github is all --visitors can't do much more that search list, search documents and view content.

**Comment by wobba** [comment, marked as answer]
There are no anonymous pages in SharePoint Online, and this question is way out of scope for this project :)

**Comment by RamBalani** [comment]
My apologies-for out of scope. 
I must have misunderstood what Powerapps portals are all about then--my bad.

![PowerApps Portal by Default are Anonymous](https://user-images.githubusercontent.com/37209670/167870720-fd0a1006-62bf-4a90-80a8-c438adeceeeb.jpg)

And this  comments:
"Anonymous access to the Power Apps Portals requires an add-on though, which is an additional US$100 extra per month, per 100K anonymous page views."
----------------------------------------------------"

**Reply by wobba** [reply]
Portals is a different product, and still nothing to do with PnP Modern Search web parts.
